models/evc.py

```diff
@@ -117,6 +117,9 @@ def __init__(self, controller, **kwargs):
         )
         self.service_level = kwargs.get("service_level", 0)
         self.circuit_scheduler = kwargs.get("circuit_scheduler", [])
+        self.flow_removed_at = get_time(kwargs.get("flow_removed_at")) or None
```

Comment:
`get_time` takes a `str` or `dict`, when loading an EVC this will pass a datetime instance, so as it is looks like it'll be discarding and set None. Could you review this and `self.updated_at` as well? 

You're using `datetime.utcnow()` when setting, which is great, so it'll probably not need to reset/replace the tz like we had to on topology https://github.com/kytos-ng/topology/blob/master/main.py#L787

Reply:
Indeed it lead to issues, please make sure to test loading EVCs and ensuring the behavior is still compatible. 

```
kytos $> 2022-12-14 15:33:59,379 - ERROR [kytos.core.napps.base] (mef_eline) NApp: <Main(mef_eline, started 139633035814592)> unhandled exception Traceback (most recent call last):
  File "/home/viniarck/repos/kytos/kytos/core/napps/base.py", line 257, in run
    self.execute()
  File "/home/viniarck/repos/napps/napps/kytos/mef_eline/main.py", line 82, in execute
    self.execute_consistency()
  File "/home/viniarck/repos/napps/napps/kytos/mef_eline/main.py", line 95, in execute_consistency
    and not circuit.has_recent_removed_flow()
  File "/home/viniarck/repos/napps/napps/kytos/mef_eline/models/evc.py", line 213, in has_recent_removed_flow
    res_seconds = (now_time - removed_time) / timedelta(seconds=1)
TypeError: can't subtract offset-naive and offset-aware datetimes
```

Reply:
I have checked again and indeed what is passed to `get_time` is a str and this function returns a datetime that is the expected data type for `updatd_at` and `flow_removed_at`.

On the other hand, I have fix both `updatd_at` and `flow_removed_at` as `now()` (link1)[https://github.com/kytos-ng/mef_eline/blob/2622aac8b1de25ef80df03a9d3e2dfec9c5877dd/models/evc.py#L202-L204] and (link2)[https://github.com/kytos-ng/mef_eline/blob/2622aac8b1de25ef80df03a9d3e2dfec9c5877dd/models/evc.py#L155].

Now it seems to be better.

Reply:
With the current DB projection looks good. Let me close this thread.